woocommerce-mondu.php

```diff
@@ -21,7 +21,7 @@
 
 defined('ABSPATH') or die('Direct access not allowed');
 
-define('MONDU_PLUGIN_VERSION', '1.1.3');
+define('MONDU_PLUGIN_VERSION', '1.1.4');
```

Comment:
**nitpick (non-blocking):** Maybe you will need to generate the translations again because they have the plugin's version on them. I am not sure how it works but I always regenerated it again after updating 😅

Reply:
I tried generating the po files again using 
wp i18n --allow-root update-po languages/mondu.pot languages/mondu-de_DE.po
comamnd, but it didn't add the new version tag to the file. Do you know how to add the version tag ?